src/infql.jl

```diff
@@ -378,3 +378,237 @@ function LazyBandedMatrices._SymTridiagonal(::Tuple{TriangularLayout{'L', 'N', P
     ev = [A[k,k+1] for k=1:m-1]
     SymTridiagonal([dv; Fill(dv[end],∞)], [ev; Fill(ev[end],∞)])
 end
+
+###
+# Experimental adaptive finite section QL
+###
+mutable struct QLFiniteSectionTau{T} <: AbstractCachedVector{T}
```

Comment:
The name `FiniteSection` is confusing as that usually means finite dimensional

Reply:
True. The name comes from the fact that we are computing QL on finite sections and then increasing size to convergence. I am happy to hear better name suggestions? QLDoublingMethod?

Reply:
`AdaptiveQLTau`

Reply:
That is what I went with after your comment :smile: